### PR TITLE
refactor(deployment): export only powerless objects (per ocap)

### PR DESCRIPTION
**Pull request by dckc**

### CODE REVIEW

File: packages/deployment/files.js [hunk 1]
```diff
@@ -1,70 +1,76 @@
...
-  readdir as rawReaddir,
-  write as rawWrite,
-  close as rawClose,
-  unlink as rawUnlink,
-  rename as rawRename,
-} from 'fs';
+import { promisify } from 'util';
 import { Readable } from 'stream';
-import { open as tempOpen } from 'temp';
 import chalk from 'chalk';
```
Comment (michaelfig) [inline packages/deployment/files.js:3]: Not sure if you care, but `'chalk'` is also ambient.  It accesses `process.env`.  Perhaps you'd like to pretend you never saw this?

File: packages/deployment/setup.js [hunk 2]
```diff
@@ -1,23 +1,28 @@
...
-export const SETUP_HOME = process.env.AG_SETUP_COSMOS_HOME
-  ? resolve(process.env.AG_SETUP_COSMOS_HOME)
-  : resolve('.');
-process.env.AG_SETUP_COSMOS_HOME = SETUP_HOME;
 
-export const playbook = (name, ...args) => {
-  const fullPath = `${SETUP_DIR}/ansible/${name}.yml`;
-  return [PLAYBOOK_WRAPPER, fullPath, ...args];
-};
+const { freeze } = Object;
```
Comment (michaelfig) [inline packages/deployment/setup.js]: We should have access to `harden` at this point.

File: packages/deployment/ag-setup-cosmos [hunk 3]
```diff
@@ -1,13 +1,32 @@
 #! /usr/bin/env node
+/* global require, module, setInterval */
 
+const fs = require('fs');
+const path = require('path');
+const temp = require('temp');
+const process = require('process');
+const { exec, spawn } = require('child_process');
+const inquirer = require('inquirer');
+const fetch = require('node-fetch');
```
Comment (michaelfig) [inline packages/deployment/ag-setup-cosmos:14]: All of these should be after `esmRequire('@agoric/install-ses')`.

**Review by michaelfig — COMMENTED** [review]
Adding comments.  I'll take a look at resolving them.

File: packages/deployment/files.js (hunk 1 above)
Comment (michaelfig) [reply packages/deployment/files.js:3]: These are not the modules you're looking for.  We'll get to it later.

File: packages/deployment/ag-setup-cosmos (hunk 3 above)
Comment (dckc) [reply packages/deployment/ag-setup-cosmos:14]: speaking of `esmRequire`... vs-code's "go to definition" feature works with it if we assign it to `require`.

b2c02e08c301

**Review by michaelfig — APPROVED** [review]
LGTMing!

**Pull request merged**